RELEASE-NOTES.md

```diff
@@ -1,5 +1,12 @@
 # Wikibase DataModel Serialization release notes
 
+## 3.0.0 (2017-02-16)
+
+* Changed the `DeserializerFactory` constructor to expect a list of Entity deserializers
+* Changed the `SerializerFactory` constructor to expect a list of Entity serializers
+* `DeserializerFactory::newEntityDeserializer` does not return default Item and Property deserializers any more
+* `SerializerFactory::newEntitySerializer` does not return default Item and Property serializers any more
```

Comment:
Personally I'm fine with keeping the two default instances. It's just that we had a lot of very similar places before, and found that in almost all cases it's more confusing than helpful to have such defaults. What happens is that a caller of such a factory function must know what it supports by default, and skip possible duplicates. Otherwise you will end with a dispatching (de)serializer with duplicate support for items and properties, and you can't even know which of the duplicates will be used.

How to solve this?